config.w32

```diff
@@ -9,8 +9,8 @@ if (PHP_MEMCACHE != "no") {
 		var dll = get_define('PHPDLL');
 		var old_conf_dir = configure_module_dirname;
 
-		if (dll.match(/^php8/) != null) {
-			configure_module_dirname = configure_module_dirname + "\\php8";
+		if (dll.match(/^src/) != null) {
```

Comment:
It seems like this change was unintentionally made in a bulk replacement

```suggestion
		if (dll.match(/^php[78]/) != null) {
```

This would check if the dll name starts with php7 or php8 - config.w32 is a JavaScript DSL

Reply:
It was intentional, but it may have unintentional side effects.
The goal is to have both php7 and php8 sources in one branch to make maintaince easier.
I'm not familiar with windows build at all, but prehpas you have an idea how this could be done without breaking anything?

Reply:
The dll is not part of the pecl-memcache project - it's the value of the environment variable `PHPDLL` on Windows, which is the basename of the dll

https://github.com/igbinary/igbinary/blob/master/config.w32#L9 is an example in another project

Reply:
Ah, I see. It was certainly unintentional then. I'll fix it.